docker-compose.yaml

```diff
@@ -1,22 +1,35 @@
-version: '3.0'
+version: '3.3'
 
 services:
-  server:
-    build: .
-    user: "$UID"
```

Comment:
afaik if you remove this line  the container will chown `development_demo`

Reply:
It will use the default container username. I don't see any issue with it.